roles/gather_information/tasks/main.yml

```diff
@@ -0,0 +1,231 @@
+---
+- name: Include vSphere creds
+  ansible.builtin.include_vars:
```

Comment:
@bardielle I think you can remove this, as you load the credentials file in playbook already.

Reply:
https://github.com/kubev2v/migration-planner/pull/7